src/map/battle.c

```diff
@@ -4470,7 +4468,7 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 
 			case TF_DOUBLE: //For NPC used skill.
 			case GS_CHAINACTION:
-				wd.type = BDT_MULTIHIT;
+				wd.type = BDT_CRIT | BDT_MULTICRIT;
```

Comment:
BDT_* is not a bitmask, is this intended? also this will break pre-re compatibility please add it under #ifdef.